snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/index/change/ComponentInactivationChangeProcessor.java

```diff
@@ -93,13 +91,21 @@ public void process(StagingArea staging, RevisionSearcher searcher) throws IOExc
 
 	private void processInactivations(StagingArea staging, RevisionSearcher searcher, Set<String> inactivatedConceptIds, Set<String> inactivatedComponentIds) throws IOException {
 		// inactivate descriptions of inactivated concepts, take current description changes into account
+		
+		ServiceProvider context = (ServiceProvider) staging.getContext();
+		ModuleIdProvider moduleIdProvider = context.service(ModuleIdProvider.class);
+		
 		if (!inactivatedConceptIds.isEmpty()) {
 			
+			
 			Multimap<String, RevisionDiff> changedMembersByReferencedComponentId = HashMultimap.create();
 			staging.getChangedRevisions(SnomedRefSetMemberIndexEntry.class).forEach(diff -> {
 				changedMembersByReferencedComponentId.put(((SnomedRefSetMemberIndexEntry) diff.newRevision).getReferencedComponentId(), diff);
 			});
 			
+			SnomedRefSetMemberIndexEntry memberForModule = staging.getChangedRevisions(SnomedRefSetMemberIndexEntry.class).findFirst().map(diff -> (SnomedRefSetMemberIndexEntry) diff.newRevision).get();
```

Comment:
It seems like this member can come from anywhere.

Reply:
As discussed, I leave the code as it is.